templates/forms/fields/checkbox/checkbox.html.twig

```diff
@@ -1,8 +1,6 @@
+{% set show_label = false %}
 {% extends "forms/field.html.twig" %}
 
-{% block label %}
-{% endblock %}
-
```

Comment:
I would approve this if you didn't remove the label block. It does no harm if not used, and might be being used by users with an extend

Reply:
@rhukster I am not sure if that would make sense. Since we are disabling the whole block, it cannot be displayed anyways. So even when extending the block (for whatever reason) it wont be displayed. Correct me, if I am wrong.

https://github.com/getgrav/grav-plugin-form/blob/develop/templates/forms/layouts/field.html.twig#L4-L20

Reply:
I need more time to review this, but need to get a release out.. so will leave this one here until I can check it with the other label related PR